cmd/snap/cmd_create_key.go

```diff
@@ -58,7 +55,7 @@ func (x *cmdCreateKey) Execute(args []string) error {
 	if keyName == "" {
 		keyName = "default"
 	}
-	if !validKeyName.MatchString(keyName) {
+	if !asserts.ValidAccountKeyName.MatchString(keyName) {
```

Comment:
Had a chat with @pedronis about this just in case there was a better solution. There isn't really, so let's leave it like this.